build.gradle

```diff
@@ -303,26 +306,58 @@ task coverage {
 task checkStyle {
 }
 
+ext {
+    guiTests = 'address/guitests/*'
+    guiUnitTests = 'address/guiunittests/*'
+    unitTests = 'address/unittests/*'
```

Comment:
Probably not. Need to tweak the settings so that it reads an array of filters instead.